tests/dummy_data/QUANT_RAMP/primaryPopulation.csv

```diff
@@ -0,0 +1,9 @@
+,Unnamed: 0,rowid,msoaiz,count_primary,count_secondary,zonei,lat,lon,osgb36_east,osgb36_north,area

```

Comment:
This file has an `Unnamed: 0` column which is typically created when using `pd.to_csv()` but not setting `index=False` to preventing it writing out the index column as a new column. Should just be careful as regular `read_csv` and `to_csv` calls will continue to create lots of Unnamed columns.

Reply:
Good point. I left it in as I wanted it to be identical to the main data used, and am loath to change it now, but will be aware in future.